src/mime.cr

```diff
@@ -1,4 +1,5 @@
 require "crystal/system/mime"
+require "./mime/**"
```

Comment:
I don't think this should require everything in the `mime` folder. `mime/multipart` should only be required when needed.
There is no interdependency between the mime registry (`MIME`) and `MIME::Multipart`.